package-template.sile-dev-1.rockspec

```diff
@@ -0,0 +1,27 @@
+package = "package-template.sile"
+version = "dev-1"
+
+description = {
+  summary = "A template for SILE packages...",
+  detailed = [[]],
+  homepage = "", -- e.g. its GitHub repo
```

Comment:
Lets go ahead and fill in the blanks throughout the rockspec with working values so it can actually be built and tested from this template repo, but document each line with a `-- TODO: ...` comment noting that it *should* be changed after initializing the project.

Reply:
Well, it can actually be built this way, I had no trouble in my attempts to built and run the test functions...